DigitalLearningSolutions.Web/Controllers/LearningMenuController/LearningMenuController.cs

```diff
@@ -72,8 +72,8 @@ public IActionResult Close()
             return RedirectToAction("Current", "LearningPortal");
         }
 
-        [Route("/LearningMenu/Section/{sectionId:int}")]
-        public IActionResult Section(int sectionId)
+        [Route("/LearningMenu/{customisationId:int}/{sectionId:int}")]
+        public IActionResult Section(int customisationId, int sectionId)

```

Comment:
Now that we have a `customisationId` in this controller method, we can add the session tracking call to `sessionService.StartOrUpdateSession(...)`. Let me know if you want a hand getting that set up 🙂